kernel/src/lib.rs

```diff
@@ -16,46 +16,44 @@ extern crate spin;
 mod lang;
 mod io;
 #[macro_use]
-mod vga;
-mod ps2_io;
-mod ps2;
+mod drivers;
 
-use vga::{VgaColor, Color};
+use drivers::vga::{VgaColor, Color};
 
-const FLOWER: &'static str = include_str!("flower.txt");
-const FLOWER_STEM: &'static str = include_str!("flower_stem.txt");
+const FLOWER: &'static str = include_str!("resources/ascii_art/flower.txt");
+const FLOWER_STEM: &'static str = include_str!("resources/ascii_art/flower_stem.txt");
 
 /// Kernel main function
 #[no_mangle]
 pub extern fn kmain() -> ! {
-    vga::WRITER.lock().fill_screen(Color::Black);
+    drivers::vga::WRITER.lock().fill_screen(Color::Black);
```

Comment:
maybe we could `use drivers::vga::WRITER` or just `use drivers::vga`

Reply:
Okay. I didn't want to `use drivers::vga::WRITER`, because we may have different writers in the future. Maybe a more descriptive field name would be beneficial?

Reply:
I reckon then we can `use drivers::vga` so then we do `vga::WRITER`

Reply:
(Until we refactor terminal)